packages/matchbox/src/components/Button/styles.js

```diff
@@ -90,25 +90,25 @@ export const colorVariant = props => {
   switch (props.buttonColor) {
     case 'orange': // To be deprecated
     case 'blue':
-      color = tokens.color_blue_700;
-      darkHoverColor = tokens.color_blue_800;
-      lightActiveColor = tokens.color_blue_200;
-      lightHoverColor = tokens.color_blue_100;
+      color = props.theme.colors.blue['700'];
+      darkHoverColor = props.theme.colors.blue['800'];
+      lightActiveColor = props.theme.colors.blue['300'];
+      lightHoverColor = props.theme.colors.blue['200'];
       break;
 
     case 'red':
-      color = tokens.color_red_700;
-      darkHoverColor = tokens.color_red_800;
-      lightActiveColor = tokens.color_red_200;
-      lightHoverColor = tokens.color_red_100;
+      color = props.theme.colors.red['700'];
+      darkHoverColor = props.theme.colors.red['800'];
+      lightActiveColor = props.theme.colors.red['300'];
+      lightHoverColor = props.theme.colors.red['200'];
       break;
 
     case 'gray':
     default:
-      color = tokens.color_gray_900;
-      darkHoverColor = tokens.color_gray_1000;
-      lightActiveColor = tokens.color_gray_200;
-      lightHoverColor = tokens.color_gray_100;
+      color = props.theme.colors.gray['900'];
+      darkHoverColor = props.theme.colors.gray['1000'];
+      lightActiveColor = props.theme.colors.gray['400'];
```

Comment:
Do blue and red also need to be 2 steps darker than they were before?

Reply:
I kind of just eyeballed it, it seemed a little too light at first. What do you think?

Reply:
I think they're good how they are